workitem/workitem_repository.go

```diff
@@ -723,6 +723,13 @@ func (r *GormWorkItemRepository) GetCountsPerIteration(ctx context.Context, spac
 	db = r.db.Raw(iterationWithWICount)
 	var res []WICountsPerIteration
 	db.Scan(&res)
+	if db.Error != nil {
+		log.Error(ctx, map[string]interface{}{
+			"spaceID": spaceID.String(),
```

Comment:
yup did in https://github.com/almighty/almighty-core/pull/1151/commits/f11d893ea23493f2a73f28b4fed578a77331e80c